gravity-forms/gw-validate-current-password.php

```diff
@@ -0,0 +1,14 @@
+<?php
+/**
+ * Gravity Wiz // Gravity Forms // Validate Current Password
+ * http://gravitywiz.com/
+ */
+// update "123" to your form ID, update "6" to your Password field ID
+add_filter( 'gform_field_validation_1639_1', function( $result, $value ) {
+	$user = wp_get_current_user();
+	if ( ! wp_check_password( $value, $user->data->user_pass, $user->ID ) ) {
```

Comment:
Add a check to see if there the the `$user` variable is truthy.

Reply:
@spivurno I've added this check